packages/open-payments/package.json

```diff
@@ -8,11 +8,15 @@
   ],
   "scripts": {
     "build:deps": "pnpm --filter openapi build",
-    "build": "pnpm build:deps && pnpm clean && tsc --build tsconfig.json",
+    "build": "pnpm build:deps && pnpm clean && tsc --build tsconfig.json && pnpm copy-files",
     "clean": "rm -fr dist/",
-    "generate:types": "npx ts-node scripts/generate-types.ts",
+    "fetch-schemas": "./scripts/fetch-schemas.sh",
+    "generate:auth-server-types": "openapi-typescript src/openapi/auth-server.yaml --output src/openapi/generated/auth-server-types.ts -t",
```

Comment:
why types over interfaces btw?

Reply:
I don't know :sweat_smile: 
`-t` isn't even an option in the version we're using...
```
  Generate TypeScript types from Swagger OpenAPI specs

  Usage
    $ openapi-typescript [input] [options]

  Options
    --help                       display this
    --output, -o                 Specify output file (default: stdout)
    --auth                       (optional) Provide an authentication token for private URL
    --headersObject, -h          (optional) Provide a JSON object as string of HTTP headers for remote schema request
    --header, -x                 (optional) Provide an array of or singular headers as an alternative to a JSON object. Each header must follow the key: value pattern
    --httpMethod, -m             (optional) Provide the HTTP Verb/Method for fetching a schema from a remote URL
    --immutable-types, -it       (optional) Generates immutable types (readonly properties and readonly array)
    --additional-properties, -ap (optional) Allow arbitrary properties for all schema objects without "additionalProperties: false"
    --default-non-nullable       (optional) If a schema object has a default value set, don’t mark it as nullable
    --prettier-config, -c        (optional) specify path to Prettier config file
    --raw-schema                 (optional) Parse as partial schema (raw components)
    --version                    (optional) Force schema parsing version
```